src/engine/contactwriter.cpp

```diff
@@ -4808,6 +4875,9 @@ QContactManager::Error ContactWriter::syncUpdate(const QString &syncTarget,
             }
         }
 
+        // create incidental local constituents to store aggregated data
+        // which does not originate from the server-side contact.
+        // XXX TODO: When does this happen such that syncUpdate() is called?
```

Comment:
I don't quite understand how an incidental contact could be created as a result of syncUpdate().  Matt, can you describe the situation where this happens?  Or, have I misunderstood what this branch is for?